packages/flutter/lib/src/material/divider_theme.dart

```diff
@@ -0,0 +1,167 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show lerpDouble;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+
+import 'theme.dart';
+
+/// Defines the visual properties of [Divider], [VerticalDivider], and dividers
+/// between [ListTile]s and between rows in [DataTable]s.
+///
+/// Descendant widgets obtain the current [DividerThemeData] object using
+/// `DividerTheme.of(context)`. Instances of [DividerThemeData]
+/// can be customized with [DividerThemeData.copyWith].
+///
+/// Typically a [DividerThemeData] is specified as part of the overall
+/// [Theme] with [ThemeData.dividerTheme].
+///
+/// All [DividerThemeData] properties are `null` by default. When null,
+/// the widgets will provide their own defaults.
+///
+/// See also:
+///
+///  * [ThemeData], which describes the overall theme information for the
+///    application.
+class DividerThemeData extends Diagnosticable {
+
+  /// Creates a theme that can be used for [DividerTheme] or
+  /// [ThemeData.dividerTheme].
+  const DividerThemeData({
+    this.color,
+    this.space,
+    this.thickness,
+    this.indent,
+    this.endIndent,
+  });
+
+  /// The color of [Divider]s and [VerticalDivider]s, also
+  /// used between [ListTile]s, between rows in [DataTable]s, and so forth.
+  final Color color;
+
+  /// The [Divider]'s width extent or the [VerticalDivider]'s height extent.
```

Comment:
Maybe leave out "extent":
The [Divider]'s width or the [VerticalDivider]'s height.

Reply:
done